File: TestSante/Classes/TransfertDTO.cs
namespace TestSante.Classes
{
    public class TransfertDTO
    {
        String idTransfert { get; set; }
        List<TransactionDTO> transactions { get; set; }
        DeviseDTO devise { get; set; }

        public TransfertDTO(String idTransfert, List<TransactionDTO> transactions, DeviseDTO devise)
        {
            this.idTransfert=idTransfert;
            this.transactions = transactions;
            this.devise = devise;
        }

        public TransfertDTO(String idTransfert, DeviseDTO devise)
        {
            this.idTransfert = idTransfert;
            this.devise = devise;
        }

        public TransfertDTO()
        {

        }
    }
}

File: BanqueApp/src/dto/TransfertDTO.java
/*
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */
package dto;

import java.io.Serializable;
import java.util.ArrayList;

/**
 *
 * @author Antsa
 */
public class TransfertDTO implements Serializable{
    String idTransfert;
    ArrayList<TransactionDTO> transactions;
    DeviseDTO devise;

    public String getidTransfert() {
        return idTransfert;
    }

    public void setidTransfert(String idTransfert) {
        this.idTransfert = idTransfert;
    }

    public ArrayList<TransactionDTO> gettransactions() {
        return transactions;
    }

    public void settransactions(ArrayList<TransactionDTO> transactions) {
        this.transactions = transactions;
    }

    public DeviseDTO getdevise() {
        return devise;
    }

    public void setdevise(DeviseDTO devise) {
        this.devise = devise;
    }

    public TransfertDTO(String idTransfert, ArrayList<TransactionDTO> transactions, DeviseDTO devise) {
        this.setidTransfert(idTransfert);
        this.settransactions(transactions);
        this.setdevise(devise);
    }

    public TransfertDTO(String idTransfert, DeviseDTO devise) {
        this.setidTransfert(idTransfert);
        this.setdevise(devise);
    }
    
    
    
    public TransfertDTO()
    {
        
    }
}
